### DISCUSSION TITLE
Welcome to OpenAndroidInstaller Discussions!

### DISCUSSION THREAD

**Discussion by tsterbak** [openandroidinstaller-dev/openandroidinstaller#328] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RonLater** [comment]
how do you enable the options for bootloader unlocked and custom recovery. tjhey are greyed out and cant use them

**Reply by tsterbak** [reply]
They'll be enabled once a device is detected.
